configure

```diff
@@ -196,6 +196,11 @@ shared_optgroup.add_option('--shared-openssl-libpath',
     dest='shared_openssl_libpath',
     help='a directory to search for the shared OpenSSL DLLs')
 
+shared_optgroup.add_option('--use-def-ca-store',
+    action='store_true',
+    dest='use_def_ca_store',
+    help='use system supplied Root CA store instead of bundled copy')
```

Comment:
This only uses a "system" CA store on Linux, one Windows, the system CA store isn't used. The name of the option should more clearly identify it as using the OpenSSL Root CA store, so people can find the docs on the location and configuration of the OpenSSL CA store.